server/service/src/requisition/common.rs

```diff
@@ -124,7 +124,7 @@ pub fn check_exceeded_max_orders_for_period(
 
             let current_orders = RequisitionRepository::new(connection).count(Some(filter))?;
 
-            Ok(current_orders > input.max_orders_per_period)
+            Ok(current_orders >= input.max_orders_per_period)
```

Comment:
If current order is equal or greater than max then it will return true and will prompt error? Or am I missing something here?

Reply:
Current orders is how many orders already exist in the DB. So being able to create one more would exceed the maximum.

Reply:
Ah right~ makes sense